Xamarin.Essentials/Permissions/Permissions.shared.cs

```diff
@@ -16,6 +16,15 @@ internal static void EnsureDeclared<TPermission>()
             where TPermission : BasePermission, new() =>
                 new TPermission().EnsureDeclared();
 
+        internal static async Task RequestAndVerifyAsync<TPermission>()
```

Comment:
Can we change this to be `EnsureGrantedAsync` or something else Ensure-y to follow the pattern of check/request/ensure? I am never a fan of "and" in method names.